### PR TITLE
feat: add support for shortcuts

### LINKED ISSUE #241: Automate Kizzy in Samsung through Routines app
**Is your feature request related to a problem? Please describe.**
Not necessarily a problem, but makes the app more convenient.

**Describe the solution you'd like**
"Modes and Routines" app in Samsung devices is capable of following a protocol when conditions are met. This could allow automatic initiation of Kizzy's "Media RPC" or "App detection" when a certain targeted app or game is opened, removing the necessity to visit Kizzy first.

**Additional context**
However to do this, Kizzy would need to have the option to start RPC in it's quick access panel, so the Routines app can fetch and trigger it.

For example here the Reminder app has the create event in it's panel so it goes detected by Routines as well:
Screenshot_20231217_211729_One UI Home

Screenshot_20231217_211441_Modes and Routines

Since Kizzy has nothing there, no event shows up on Routines to trigger.

I'm aware that a Kizzy button is also available in the notification panel which makes me think this is possible to pull off as well.
Thank you.

### LINKED ISSUE #270: Tasker/automation tools in general support
Hello, could you please add support for automation apps ? (Tasker, Macrodroid, etc... they all use the same communication system i think)
So they can set and enable the custom status for example or modify some settings
Thanks !

### PR DESCRIPTION
**Pull request by kyrie25**
Support for app shortcuts, used for automation apps

image
image

Resolves #241
Resolves #270

### CODE REVIEW

File: common/resources/src/main/res/drawable/shortcuts_apps.xml
Comment (dead8309) [inline common/resources/src/main/res/drawable/shortcuts_apps.xml:1]: Any specific reason for creating new icon instead of using previous ones ?

Comment (kyrie25) [reply common/resources/src/main/res/drawable/shortcuts_apps.xml:1]: - Android auto fills white background on holding app icon, while current icon has white strokes
- Current icon doesn't have paddings, which makes it the icon content too big for the hover menu, causing clips

Comment (dead8309) [reply common/resources/src/main/res/drawable/shortcuts_apps.xml:1]: Oh then it's okay

**Comment by ragxxv** [comment]
https://github.com/user-attachments/assets/eddb0016-148e-4361-8f0d-4cfffea7787b

Is the toast message by Kizzy kicking me out of the app each time? This practically makes the media app unusable with automation.

**Pull request merged**